src/drivers/px4fmu/fmu.cpp

```diff
@@ -1057,20 +1062,29 @@ PX4FMU::cycle()
 				if (_poll_fds[poll_id].revents & POLLIN) {
 					orb_copy(_control_topics[i], _control_subs[i], &_controls[i]);
 
-					/* main outputs */
+#if defined(DEBUG_BUILD)
+
+					static int main_out_latency = 0;
+					static int sum_latency = 0;
+					static uint64_t last_cycle_time = 0;
+
 					if (i == 0) {
-//						main_out_latency = hrt_absolute_time() - _controls[i].timestamp - 250;
```

Comment:
this code was commented out; I modified it to measure and print latency from control to publishing actuator outputs, but it needs a code inspection and bracketing with DEBUG_BUILD